contrib/test.sh

```diff
@@ -35,8 +35,8 @@ ${dbxcli} cp ${d}/dbxcli ${d}/dbxcli-new
 echo "Testing revs"
 rev=$(${dbxcli} revs ${d}/dbxcli)
 
-echo "Testing mv"
-${dbxcli} mv ${d}/dbxcli ${d}/dbxcli-old
+echo "Testing updated mv"
```

Comment:
Let's just keep the old test and add a new one so we test both the trailing slash and without slash

Reply:
Ok! Updated test.sh with both tests.